src/Components/src/Microsoft.AspNetCore.Components/Rendering/Renderer.cs

```diff
@@ -240,6 +380,23 @@ private void ProcessRenderQueue()
             }
         }
 
+        private bool TryGetRenderQueueEntry(out RenderQueueEntry entry)
```

Comment:
Possible rename: `TryDequeueRenderQueueEntry` for clarity.

Reply:
Done